### PR TITLE
feat: support exclude atypes in atomic model

### PR DESCRIPTION
**Pull request by wanghan-iapcm**
With this pr, we do not need to implement the exclude types for each descriptor and fitting.

### CODE REVIEW

**Review by njzjz — APPROVED** [review]

File: deepmd/utils/argcheck.py
Comment (iProzd) [inline deepmd/utils/argcheck.py:870]: Add (Supported Backend: PyTorch) after #3342 merged.

**Review by iProzd — APPROVED** [review]

File: deepmd/utils/argcheck.py
Comment (njzjz) [reply deepmd/utils/argcheck.py:870]: I clicked the merge button..

**Pull request merged**
